ngraph/frontend/onnx_import/src/op/global_max_pool.cpp

```diff
@@ -29,7 +31,25 @@ namespace ngraph
             {
                 OutputVector global_max_pool(const Node& node)
                 {
-                    return pooling::GlobalPoolingFactory(node).make_max_pool();
+                    auto data = node.get_ng_inputs()[0];
+                    auto data_rank = data.get_partial_shape().rank();
+
+                    NGRAPH_CHECK(data_rank.is_static(),
+                                 "The input data tensor's rank has to be known (static)");
+
+                    auto data_rank_value = data_rank.get_length();
+
+                    NGRAPH_CHECK(data_rank_value > 2,
+                                 "The input data tensor's rank has to be greater than 2."
+                                 "Provided data rank is: ",
+                                 data_rank_value);
+
+                    uint64_t data_spatial_rank = data_rank_value - 2;
+                    auto reduce_axes_vector = std::vector<std::int64_t>(data_spatial_rank);
+                    std::iota(reduce_axes_vector.begin(), reduce_axes_vector.end(), 2);
+                    auto reduce_axes = default_opset::Constant::create(
+                        element::i64, Shape{data_spatial_rank}, reduce_axes_vector);
```

Comment:
Please add a comment to explain what the expected value of `reduce_axes` is. What are you doing in these lines?